writeit/settings.py

```diff
@@ -376,23 +376,23 @@
     },
     # Resyncs popit every week
     'resync-popit-apis-every-week': {
-        'task': 'nuntium.tasks.pull_from_popolo_json',
+        'task': 'nuntium.tasks.update_all_instances',
         'kwargs': {
             'periodicity': '1W'
         },
         'schedule': crontab(hour=5, minute=30, day_of_week=1),
     },
     # Resyncs popit every day
     'resync-popit-apis-every-day': {
-        'task': 'nuntium.tasks.pull_from_popolo_json',
+        'task': 'nuntium.tasks.update_all_instances',
         'kwargs': {
             'periodicity': '1D'
         },
         'schedule': crontab(hour=5, minute=30),
     },
```

Comment:
Is there any need for these if we have the twice a day one? Is this used to make the schedule configurable in admin?

Reply:
@jbothma that's correct - it is to configure how often a source should be updated in admin. If you go to the site's settings in WIP, then click on "Data sources" and then edit the source, this is what you'll see:

![2020-06-09-15:45](https://user-images.githubusercontent.com/4767109/84155225-640b3c00-aa68-11ea-823a-7a05b8b204be.png)